cost/oracle/oracle_cbi/oracle_cbi.pt

```diff
@@ -0,0 +1,478 @@
+name "Oracle Cloud Common Bill Ingestion"
+rs_pt_ver 20180301
+type "policy"
+short_description "Downloads cost reports from Oracle Cloud (OCI) and then uploads them to a Flexera CBI endpoint. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/oracle/oracle_cbi) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "15 minutes"
+info(
+  version: "2.0",
+  provider: "Oracle"
+)
```

Comment:
should have a `policy_set` and `service` in the info block

Not sure whether service applies here but maybe policy_set should be "Common Bill Ingest".
@cdwilhelm @bryankaraffa what are your thoughts?

Reply:
@cdwilhelm @bryankaraffa Thoughts on this? I can add them and leave them blank or put values in them.

Reply:
policy_set Common Bill Ingest is good.  
I'm not sure service applies either.  you can leave blank or not include.  we can add it later if needed.

Reply:
Agree on policy_set = "Common Bill Ingest"